frappe/patches/v14_0/set_document_expiry_default.py

```diff
@@ -0,0 +1,5 @@
+import frappe
+
+def execute():
+	frappe.db.set_value("System Settings", "System Settings", "document_share_key_expiry", 30)
+	frappe.db.set_value("System Settings", "System Settings", "allow_older_web_view_links", 1)
```

Comment:
Pass dict to reduce query 😅

Reply:
Done... that usage does not feel intuitive though 😅